### PR TITLE
Thai/project outline

### PR DESCRIPTION
**Pull request by athai**
WIP of draft docs. Images will be added soon.

### CODE REVIEW

File: docs/source/outline.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+.. note::
+
+  Alongside **Users**, basic CRUD will need to be implemented for **Resources**
+  and **Map/Sites** as well.
+
+Analyze/Learn
+-------------
+
+The Analyze/Learn page will also redirect the user to ``Sites``.
```
Comment (matthewrsj) [inline docs/source/outline.rst]: Does `Analyze/Learn` have anything else, or is it just another name for the `Sites` page?

**Comment by matthewrsj** [comment]
@athai I think this looks good for a good draft design doc. This will obviously change as we develop and meet with the client more. +1 for a well-written draft.

File: docs/source/outline.rst (hunk 1 above)
Comment (athai) [inline docs/source/outline.rst]: I think for now, ``Analyze/Learn`` is just another name for the ``Sites`` page.

**Comment by athai** [comment]
@LyonesGamer @leian7 Is there anything in this document that seems unclear to either of you?

**Comment by leian7** [comment]
It's clear, +1

**Comment by LyonesGamer** [comment]
Seems pretty clear to me. +1

**Comment by matthewrsj** [comment]
+1

Note: Github's new `view a subset of changes` feature is really nice.

**Comment by leian7** [comment]
+1

**Pull request merged**